src/sparseml/openpifpaf/train.py

```diff
@@ -228,5 +264,46 @@ def main():
     trainer.loop(train_loader, val_loader, start_epoch=start_epoch)
 
 
+def _load_managers_from_checkpoint(
```

Comment:
Let's update function name since it also includes weight loading

Reply:
Updated